lua/lspinstall/servers/java.lua

```diff
@@ -0,0 +1,48 @@
+local config = require'lspconfig'.jdtls.document_config
+require'lspconfig/configs'.jdtls = nil -- important, immediately unset the loaded config again
+
+config.default_config.cmd[1] = {"./jdtls.sh", vim.fn.expand('$HOME/.local/share/nvim/lspinstall/java/workspace/') .. vim.fn.fnamemodify(vim.fn.getcwd(), ':p:h:t')}
+
+return vim.tbl_extend('error', config, {
+  install_script = [[
+  curl -L -o - http://download.eclipse.org/jdtls/snapshots/jdt-language-server-latest.tar.gz | tar xvz
+  curl -L -o lombok.jar https://projectlombok.org/downloads/lombok.jar
+cat << EOF > jdtls.sh
+#!/usr/bin/env bash
+OS=$(uname)
+
+case \$OS in
+  Linux)
+    CONFIG="$HOME/.local/share/nvim/lspinstall/java/config_linux"
```

Comment:
Don't hardcode these paths. The install script is executed with the CWD set to `.../lspinstall/java/`, if outside of `jdtls.sh` you can just use `./config_linux` or `$(pwd)/config_linux` if jdtls needs absolute paths.